elyses-enchantments/enchantments.js

```diff
@@ -50,9 +48,7 @@ export function insertItemAtTop(cards, newCard) {
  * @returns {number[]} the cards without the removed card
  */
 export function removeItem(cards, position) {
-  const newCards = Array.from(cards);
-  newCards.splice(position, 1);
-  return newCards;
+  return [...cards.slice(-cards.length,position),...cards.slice(position+1)];
```

Comment:
Parameters of `Array#slice` are `indexFrom` and `indexTo` (exclusive).

```js
const arr = [ 1, 2, 3, 4, 5 ]; // => <0> 1 <1> 2 <2> 3 <3> 4 <4> 5
arr.slice(2, 4) // =>                             ^           ^    => [ 3, 4 ]

// not want <2>                       ^-----------^     ^...............
const no2 = [...arr.slice(0, 2), ...arr.slice(3)];
//                                   <-5>                    <-1>
```
Why use `-cards.length` ?